ChatroomUIKit/build.gradle.kts

```diff
@@ -65,7 +65,8 @@ dependencies {
     implementation("androidx.lifecycle:lifecycle-runtime-ktx:2.6.1")
     implementation("androidx.lifecycle:lifecycle-viewmodel-compose:2.6.1")
     implementation("io.coil-kt:coil-compose:2.4.0")
-    api(project(mapOf("path" to ":ChatroomService")))
+//    api(project(mapOf("path" to ":ChatroomService")))
+    api("io.hyphenate:hyphenate-chat:4.2.0")
```

Comment:
I am assuming this is placeholder for beta, but we need to make sure this is updated before pushing to production/GA